### PR TITLE
Make README suitable for users

### PR DESCRIPTION
**Pull request by jimmykarily**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,80 @@
...
+and runs Builds for a project specified by APP_ID and APP_SECRET environment
+variables.
 
-## Manager
+## Status
 
-This is the main thread. It keeps the list of jobs filled and passes jobs to
-the worker thread for execution. Its work in pseudocode:
+This is the Go version of the Agent originally written as a [Ruby gem](https://github.com/testributor/testributor_gem).
+It is currently in Beta and it is missing some functionality related to Bare repository projects. It should working for
```
Comment (pakallis) [inline README.md]: It should work

File: README.md [hunk 2]
```diff
@@ -1,34 +1,80 @@
...
 
-  select {
-    - send jobs to the job channel (read by the worker)
-    - read fetched jobs from the jobs channel (written by the go routine spawned previously)
-      write the jobs to the job list
-  }
-}
+This program's functionality is directly related to the functionality offered by
+Testributor. For this reason, changes on Testributor might trigger changes on the
+Agent as well. For this reason, popular versioning schemes (schemantic?) might not
```
Comment (pakallis) [inline README.md]: Perhaps `semantic`?

File: README.md (hunk 1 above)
Comment (jimmykarily) [inline README.md]: Changed

File: README.md (hunk 2 above)
Comment (jimmykarily) [inline README.md]: right

**Comment by pakallis** [comment]
:+1:

**Pull request merged**